.github/workflows/e2e-manual-icad.yaml

```diff
@@ -0,0 +1,55 @@
+name: Manual E2E (Icad)
+on:
+  # when https://github.com/community/community/discussions/11795 is resolved
+  # we will be able to dynamically build up the list of valid inputs.
+  # for now this needs to be manual.
+  workflow_dispatch:
+    inputs:
+      test-entry-point:
+        description: 'Test entry point'
+        required: true
+        type: choice
+        options:
+          - TestInterchainAccountsTestSuite
+      chain-image:
+        description: 'The image to use for chain A'
+        required: true
+        type: string
+        default: "ghcr.io/cosmos/ibc-go-icad"
+      chain-a-tag:
+        description: 'The tag to use for chain A'
+        required: true
+        type: choice
+        default: master
+        options:
+          - master
+          - v0.3.2
+          - v0.2.2
+          - v0.1.2
```

Comment:
these versions all include [this fix](https://github.com/cosmos/interchain-accounts-demo/pull/111) to allow for cross version testing